doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,259 @@
+
+.. code-block:: text
+
+  subsection insertion info
+    set insertion method                               = volume
+    set inserted number of particles at each time step = 10000
+    set insertion frequency                            = 9000
+    set insertion box points coordinates               = 0.015, -0.00325, -0.00325: 0.035, 0.00325, 0.00325
+    set insertion distance threshold                   = 1.2
+    set insertion maximum offset                       = 0.05
+    set insertion prn seed                             = 19
+    set insertion direction sequence                   =  1, 2, 0
+  end
+
+
+Solid Object
+~~~~~~~~~~~~
+
+Two mesh files are necessary to reproduce the granuheap example; ``cylinder.msh`` and ``support.msh``. The cylinder is 0.02 tall with a radius of 0.005 and the support is 0.005 tall and has the same diameter as the cylinder.
+
+.. figure:: images/meshfiles.png
+    :width: 400
+    :alt: Mesh
+    :align: center
+
+We set the cylinder's translational velocity to :math:`0.05 m/s` after the particles were loaded, thus when the simulation time is higher than 1.6s. 
+
+.. code-block:: text
+
+  subsection solid objects
+    subsection solid surfaces
+      set number of solids = 2
+      subsection solid object 0
+        subsection mesh
+          set type                   = gmsh
+          set file name              = cylinder.msh
+          set simplex                = true
+        end
+        subsection translational velocity
+          set Function expression = if (t>1.6, 0.05, 0) ; 0 ; 0
+        end
+      end
+      subsection solid object 1
+        subsection mesh
+          set type                   = gmsh
+          set file name              = support.msh
+          set simplex                = true
+        end
+      end
+    end
+  end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation runs for 2.2 seconds in real time. We output the simulation results in every 1000 iterations.
+
+.. code-block:: text
+
+  subsection simulation control
+    set time step         = 7.54e-6
+    set time end          = 2.2
+    set log frequency     = 1000
+    set output frequency  = 1000
+    set output path       = ./output/
+    set output name       = granuheap
+    set output boundaries = true
+  end
+    
+-----------------------
+Running the Simulation
+-----------------------
+
+A simulation with one set of values for the ``rolling friction`` and the ``surface energy`` can be launched using the following command:
+
+.. code-block:: text
+  :class: copy-button
+
+   mpirun -np 8 lethe-particles granuheap.prm
+
+
+.. note::
+  This example needs a simulation time of approximately 5 hours on 12 processors using an AMD Ryzen 9 5900x 12-core processor x 24.
```

Comment:
what do you mean x24?

Reply:
@blaisb I think it's the number of thread. I made a copy paste from my settings but I believe it's not a necessary information. I'll remove the x 24.

Reply:
yeah just say 12 because we don't use the threads, only the physical cores :)